libbeat/docs/release-notes/breaking/breaking-7.8.asciidoc

```diff
@@ -12,7 +12,13 @@
 
 //tag::notable-breaking-changes[]
 [float]
+==== Known problem with Kafka output
```

Comment:
Not sure what is better but I think the term `known issue` instead of `known problem` is more common in such cases. For instance: https://kind.sigs.k8s.io/docs/user/known-issues/

Reply:
+1

Reply:
TBH, I don't think it really matters because we don't use these terms consistently across Elastic docs. Since you have a preference, though, I'm happy to put up a PR. Also the terminology does map nicely to GitHub where we track all "issues." Thanks for your feedback!